src/mappings/Competition/mapping.ts

```diff
@@ -100,6 +140,31 @@ export function handleNewVote(event: NewVote): void {
       suggestion.save();
     }
     vote.save();
+
+    let competition = Competition.bind(event.address);
+    competitionProposal.winningSuggestions = [];
+    let winningSuggestions = competitionProposal.winningSuggestions;
+    debug(competitionProposal.suggestions.toString() + '');
+    for (let i = 0; i < competitionProposal.suggestions.length; i++) {
```

Comment:
how about just sorting the suggestions here ?

Reply:
I'm not sure sorting is possible at all. If it is then the problem will be supporting tie cases.